kml/dom/xsd.h no longer installed

The gdal & goldencheetah Debian packages fail to build with libkml 1.3.0-rc1:

```
In file included from /usr/include/kml/dom/element.h:40:0,
                 from /usr/include/kml/dom/object.h:31,
                 from /usr/include/kml/dom/abstractview.h:29,
                 from /usr/include/kml/dom/kmldom.h:29,
                 from /usr/include/kml/dom/kml_cast.h:32,
                 from /usr/include/kml/dom.h:33,
                 from /usr/include/kml/convenience/convenience.h:33,
                 from KmlRideFile.cpp:31:
/usr/include/kml/dom/kml22.h:37:25: fatal error: kml/dom/xsd.h: No such file or directory
```

`kml/dom/xsd.h` was not installed by the old Autotools builds, but shouldn't have been removed in b34517b3545cb6e91563675c0a5b7a5112faf4b2 . Since 73563c403870139d622fe24ae05f8300ecb704a1 `kml/dom/xsd.h` should be installed too.

--- a/src/kml/dom/CMakeLists.txt
+++ b/src/kml/dom/CMakeLists.txt
@@ -46,6 +46,7 @@ set(INCS
   gx_tour.h
   vec2.h
   xal.h
+  xsd.h
   visitor.h
   visitor_driver.h)
 

